packages/bpk-component-content-cards/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "bpk-component-content-cards",
+  "version": "1.1.13",
```

Comment:
Can you set this to 0.0.1? Just for when we do the release we can just mark as major

This isn't a fault of this PR but just our automation which I will raise a PR to fix